boxoffice/models/discount_policy.py

```diff
@@ -43,6 +45,9 @@ class DiscountPolicy(BaseScopedNameMixin, db.Model):
     percentage = db.Column(db.Integer, nullable=True)
     # price-based discount
     is_price_based = db.Column(db.Boolean, default=False, nullable=False)
+    discount_code_base = db.Column(db.Unicode(20), nullable=True)
```

Comment:
This column was added in the first [init migration](https://github.com/hasgeek/boxoffice/blob/master/alembic/versions/4ffee334e82e_init_models.py). The reference to the column in the model was removed since it wasn't of use until signed codes were implemented.